devops/Makefile

```diff
@@ -291,6 +291,12 @@ init-redis-cluster:
 	REDIS_NODES=$$(oc get pods -n $(NAMESPACE) -l app=redis -o jsonpath='{range .items[*]}{.status.podIP}:6379 ' | sed 's/ :6379 $$/ /'); \
 	oc exec -n $(NAMESPACE) -it redis-0 -- redis-cli -a $$REDIS_PASSWORD --cluster create $$REDIS_NODES --cluster-replicas 1
 
+# Command to tell one Redis node to connect to another node and join the cluster. 
+# This is typically done when you are setting up a Redis Cluster or adding new nodes to an existing cluster to ensure they are aware of each other and can communicate effectively.
```

Comment:
Is there any harm in executing the command at the wrong moment?
For instance, if Redis is not working properly, is it ok to execute the command as a first attempt to recover the Cluster?

Reply:
No i have tried it, it doesnt do anything if everything is working

Reply:
Good to know 😉